File: backend/src/main/java/NetworkStandard/Messages/GameLeaveMessage.java
package NetworkStandard.Messages;

import NetworkStandard.MessageContainer.MessageContainer;
import NetworkStandard.MessageTypeEnum.MessageTypeEnum;

import java.util.Date;
import java.util.UUID;

/**
 * Message send by the client to inform server about client leaving the game.
 *
 * @author Standardisierungsdokument & Sedat Qaja
 */
public class GameLeaveMessage extends MessageContainer {
    public GameLeaveMessage(UUID playerId, MessageTypeEnum type, Date creationDate, String debugMessage) {
        super(playerId, MessageTypeEnum.GAME_LEAVE, creationDate, debugMessage);
    }
}

File: dummy-ai-client/NetworkStandard/Messages/game_leave_message.py
from NetworkStandard.MessageContainer.message_container import MessageContainer
from NetworkStandard.MessageTypeEnum.messagetype_enum import MessageTypeEnum

import uuid


class GameLeaveMessage(MessageContainer):

    def __int__(self, player_Id, type_enum, creation_date, debug_message):
        super().__init__(player_Id, type_enum, creation_date, debug_message)
